packages/deployer/terraform/variables.tf.workaround

```diff
@@ -0,0 +1,24 @@
+variable "region" {
+  type    = string
+  default = "<UPDATE_REGION>"
```

Comment:
Terraform only works correctly when the region is hardcoded in this file. That's why the deployer copies this file and replaces `<UPDATE_REGION>` with the correct value each time it starts.